plugin/hashicorp_key_management/hashicorp_key_management_plugin.cc

```diff
@@ -562,7 +562,7 @@ int HCData::curl_run (const char *url, std::string *response,
     else if (is_error)
     {
       my_printf_error(ER_UNKNOWN_ERROR, PLUGIN_ERROR_HEADER
-                      "Hashicorp server error: %d, response: %s",
+                      "Hashicorp server error: %ld, response: %s",
                       ME_ERROR_LOG_ONLY | ME_WARNING, http_code, res);
```

Comment:
Why is `http_code` a `long` anyways? Even a `signed short` is enough to fit four decimal digits.